src/components/ProjectionWrapper.vue

```diff
@@ -32,6 +32,12 @@
           <datepicker v-model="projectionDate" name="projectionDate" placeholder="Projection Date" format="yyyy-MM-dd"/>
         </div>
       </div>
+      &ensp;
+      <b>Priorities:</b>
+      <div v-for="option in goals">
```

Comment:
Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key